### DISCUSSION TITLE
Some insights about Giscus html/ editing css

### DISCUSSION THREAD

**Discussion by iacchus** [giscus/giscus#131] [Ideas]
I've spent the last few hours tinkering with the Giscus css and had some insights:

I think if giscus had a mode with the (1.) css framework stripped off which had a (2.) simple skelenton html all namespaced with the `.gsc-` classes and (3.) with a simple base css, I think would encourage people to develop different and beautiful templates with more freedom, only by editing the css.

Maybe with a default `.gsc-` classes and something like empty `.gsc-custom-` classes with preemptiness in the markup, so that them would override the default ones?

For example I am having to add some selectors to edit some things (like `div:nth-child(2)`), to unset framework classes, to add some css's `!important` overrides etc.

Maybe then we could make a portifolio repository of different themes with images at `github.io` so that people could choose from.

These are some suggestions that happened in my mind :), thanks for the software!

**Comment by laymonage** [comment]
Hey @iacchus, thanks for the insights.

Yeah, I am aware that the CSS customization is still very limited and hard to accomplish without resorting to `!important` and writing overly specific selectors. This is partly related to #41. I still haven't decided where to go with this, but maybe the initial step would be to drop Tailwind and use plain CSS. I'll try to look into this next weekend (or the next, or... :slightly_smiling_face:)

**Comment by laymonage** [comment]
Hey @iacchus, I tried to improve the customizability in #158. I hope that helps!
